deluge/tests/test_ui_entry.py

```diff
@@ -31,6 +28,13 @@
 from .basetest import BaseTestCase
 from .daemon_base import DaemonBase
 
+try:
```

Comment:
Use a windows check to exclude importing instead of a try..except. We want the test to error if there are any issues importing these modules.